README.md

```diff
@@ -47,109 +60,118 @@ Here is the complete list of pre-requisites:
 - Access to OpenShift 4 cluster
 
 ### Build
+To build the operator use:
 ```
 make build
 ```
 ### Test
-* run unit test:
-```
-make test
-```
-* run e2e test:
-For running e2e tests, have minishift started.
-```
-make test-e2e-local
-```
-> Note: e2e test will deploy operator in project `devconsole-e2e-test`, if your tests timeout and you wan to debug:
-> - oc project devconsole-e2e-test
-> - oc get deployment,pod
-> - oc logs pod/devconsole-operator-5b4bbc7d-4p7hr
+* To run unit test use:
+    ```
+    make test
+    ```
+* To run e2e test:
+
+    Start Minishift and run:
+    ```
+    make test-e2e-local
+    ```
+
+    **Note:**
+The e2e test deploys the operator in the project `devconsole-e2e-test`. If your tests timeout and you want to debug, run:
+    ```
+    oc project devconsole-e2e-test
+    oc get deployment,pod
+    oc logs pod/devconsole-operator-5b4bbc7d-4p7hr
+    ```
+
 
 ## Deployment
 
-### Set up Minishift (one-off)
-* create a new profile to test the operator
-```
-minishift profile set devconsole
-```
-* enable the admin-user add-on
-```
-minishift addon enable admin-user
-```
-* start the instance
-```
-make minishift-start
-```
-> NOTE: this setup should be deprecated in favor of [OCP4 install]().
+**Prerequisites:**
 
-### Deploy the operator in dev mode
+Set up Minishift (a one time task):
 
-* In dev mode, simply run your operator locally:
-```
-make local
-```
-> NOTE: To watch all namespaces, `APP_NAMESPACE` is set to empty string. 
-If a specific namespace is provided only that project will watched. 
-As we reuse `openshift`'s imagestreams for build, we need to access all namespaces.
+1. Create a new profile to test the operator.
+    ```
+    minishift profile set devconsole
+    ```
+1. Enable the admin-user add-on.
+    ```
+    minishift addon enable admin-user
+    ```
+1. Start the instance.
+    ```
+    make minishift-start
+    ```
+**NOTE:** Eventually this setup will be deprecated in favor of [OCP4 installation]().
```

Comment:
@Preeticp Do you mean CRC? And the link is not given.

Reply:
@baijum I have not changed this, it is as mentioned in the original, see line 83 in the original doc. CRC makes sense to me, but we might need to check with the original author (Shoubhik?) of this readme to see if they had any other specific thing in their mind. 
For CRC, I can add this link: https://github.com/code-ready/crc, alternatively let me know if you have any other link, I can add it.

Reply:
Changed to CRC with link